src/Model/Event.php

```diff
@@ -23,34 +43,60 @@ public function getId(): int
     }
 
     /**
-     * @param mixed $id
-     *
-     * @return Event
+     * @param int $id
      */
-    public function setId($id): Event
+    public function setId(int $id): void
```

Comment:
ton entité a parfois une fluent Interface et parfois non... Ca n'est pas très cohérent. Il faut décider l'un ou l'autre. La fluent Interface c'est le fait de retourner $this dans les setter afin de pouvoir chainer les méthodes si nécessaire.